src/System.Threading.Timer/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      net463|AnyOS;
```

Comment:
See https://gist.github.com/ericstj/cb427010dbcb416fda82a8884143c32f and https://github.com/dotnet/corefx/blob/dev/eng/src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs